tests/testthat/test-between_within.R

```diff
@@ -0,0 +1,80 @@
+# h_df_1d_bw ----
+
```

Comment:
these tests look good, thanks!
please also add integration type tests for the use in the other functions (i.e. upstream calls that ask for between within type ddf)